testkit/src/main/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -520,7 +520,9 @@ public class SqlParserTest {
       "TEMPORARY",                     "92", "99",
       "THEN",                          "92", "99", "2003", "2011", "2014", "c",
       "TIME",                          "92", "99", "2003", "2011", "2014", "c",
+      "TIME_TRUNC", // BigQuery
```

Comment:
I'm surprised there are new keywords. We needed a new keyword for `ILIKE` because it's infix, but these can just be function names.

Reply:
This is me following an example commit too closely.